### PR TITLE
i/6497: Add tooltips to the table and table cell properties buttons

### PR DESCRIPTION
**Pull request by panr**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Added missing tooltips to the table and table cell properties buttons. Closes ckeditor/ckeditor5#6497.

### CODE REVIEW

File: src/ui/colorinputview.js [hunk 1]
```diff
@@ -174,6 +175,7 @@ export default class ColorInputView extends View {
 		} );
 
 		dropdown.buttonView.children.add( colorPreview );
+		dropdown.buttonView.tooltip = t( 'Color picker' );
```
Comment (jodator) [inline src/ui/colorinputview.js]: Could you please add test for these? Something changed and we havn't documented it with tests.

Comment (panr) [reply src/ui/colorinputview.js:178]: Added.

File: src/ui/utils.js [hunk 2]
```diff
@@ -241,7 +241,8 @@ export function fillToolbar( { view, icons, toolbar, labels, propertyName, nameT
 
 		button.set( {
 			label: labels[ name ],
-			icon: icons[ name ]
+			icon: icons[ name ],
+			tooltip: labels[ name ]
```
Comment (jodator) [inline src/ui/utils.js:245]: @panr Those changes needs tests also ;) Sorry for not marking that earlier.

Comment (panr) [reply src/ui/utils.js:245]: Oh, sorry. Classic me. It's not your fault 😎

Comment (panr) [reply src/ui/utils.js:245]: @jodator added ;-)

**Pull request merged**